azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/StorageService.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.storage;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+
+/**
+ * Azure storage service types.
+ */
+@LangDefinition
+public class StorageService {
```

Comment:
since this is a hand-written expandable enum with a single string mapping, why not derive from [`ExpandableStringEnum`](https://github.com/Azure/azure-sdk-for-java/blob/master/azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/ExpandableStringEnum.java)?